addon/components/bootstrap-checkbox-button.js

```diff
@@ -9,7 +9,7 @@ export default Ember.Component.extend({
   }),
 
   didReceiveAttrs() {
-    let selected = this.get('selected');
+    let selected = Ember.makeArray(this.get('selected'));
     if(Ember.isEmpty(selected)) { selected = []; }
```

Comment:
Also, what you probably want here is `Ember.A(this.get('selected'))` as `makeArray` is privatel